test/cmdlineTests.sh

```diff
@@ -66,11 +72,22 @@ function ask_expectation_update()
 {
     if [ $INTERACTIVE ]
     then
+        if [ $AUTOUPDATE = "true" ]
+        then
+            echo "$newExpectation" > $expectationFile ;
+            return
+        fi
         local newExpectation="${1}"
         local expectationFile="${2}"
+        if [[ "$AUTOUPDATE" == "true" ]]
+        then
+            echo "$newExpectation" > $expectationFile ;
+            return
+        fi
         while true;
         do
-            read -p "(u)pdate expectation/(q)uit? "
+            read -N 1 -p "(u)pdate expectation/(q)uit? "
+            echo
```

Comment:
Note to self: Read exactly one character (no delimiters) and add a new line after user input.